components/Panel/ClaimPanel/V1Rewards.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, LoadingSkeleton } from "components";
+import { BigNumber } from "ethers";
+import { formatNumberForDisplay } from "helpers";
+import Bell from "public/assets/icons/bell.svg";
+import styled from "styled-components";
+
+interface Props {
+  totalRewards: BigNumber;
+  onWithdraw: () => void;
+  isWithdrawing: boolean;
+}
+export function V1Rewards({ totalRewards, onWithdraw, isWithdrawing }: Props) {
```

Comment:
this presentational component gives the user information about their v1 rewards and lets them click the claim button to withdraw